code/modules/power/cable.dm

```diff
@@ -589,30 +589,15 @@ var/global/list/datum/stack_recipe/cable_coil_recipes = list(
 	if(istype(W, /obj/item/stack/cable_coil))
 		var/obj/item/stack/cable_coil/C = W

```

Comment:
Add a comment around here that indicates that merging is handled in the parent proc, please.